eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -118,3 +121,11 @@ services:
             - [setFragmentPath, [%fragment.path%]]
             - [setSiteAccess, [@ezpublish.siteaccess]]
         abstract: true
+
+    ezpublish.converter.repository:
```

Comment:
Given the name of the service, it feels like it would make sense to inject the repository directly into it, would it not ? Unless you don't intend to handle other types of Repository arguments (Section, User), you will end up adding all services as dependencies.